src/actions/arcActions.ts

```diff
@@ -1173,6 +1160,29 @@ export function onRedeemEvent(proposalId: string) {
   };
 }
 
+export enum RewardType {
```

Comment:
this should go in arcReducer.ts (actually i desperately want to create a types or models file)

Reply:
Sure. And we'll make the refactoring a seperate PR.

Reply:
https://github.com/daostack/alchemy/pull/155/commits/14a02fc8da2301b4506f6ffaca6419dd700d0cbc